<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sagem MORPHO Terminal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resource.qrc">
    <normaloff>:/application.ico</normaloff>:/application.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="action_SAG0"/>
   <addaction name="action_SAG1"/>
   <addaction name="separator"/>
   <addaction name="actionBreak"/>
   <addaction name="action_BDR0"/>
   <addaction name="action_BDR1"/>
   <addaction name="separator"/>
   <addaction name="action_TRACE"/>
   <addaction name="separator"/>
   <addaction name="action_IDF0"/>
   <addaction name="action_IDF1"/>
   <addaction name="action_IDC0"/>
   <addaction name="action_IDC1"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="morpho.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="morpho.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="morpho.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="morpho.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="morpho.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBreak">
   <property name="text">
    <string>$BRK</string>
   </property>
   <property name="toolTip">
    <string>Send break</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_SAG0">
   <property name="text">
    <string>$SAG0</string>
   </property>
   <property name="toolTip">
    <string>Internal reader</string>
   </property>
  </action>
  <action name="action_SAG1">
   <property name="text">
    <string>$SAG1</string>
   </property>
   <property name="toolTip">
    <string>External reader</string>
   </property>
  </action>
  <action name="action_BDR0">
   <property name="text">
    <string>$BDR0</string>
   </property>
   <property name="toolTip">
    <string>Baudrate 9600</string>
   </property>
  </action>
  <action name="action_BDR1">
   <property name="text">
    <string>$BDR1</string>
   </property>
   <property name="toolTip">
    <string>Baudrate 115200</string>
   </property>
  </action>
  <action name="action_TRACE">
   <property name="text">
    <string>$TRACE</string>
   </property>
   <property name="toolTip">
    <string>Trace statistic</string>
   </property>
  </action>
  <action name="action_IDF0">
   <property name="text">
    <string>$IDF0</string>
   </property>
   <property name="toolTip">
    <string>IDENTIFY</string>
   </property>
  </action>
  <action name="action_IDC0">
   <property name="text">
    <string>$IDC0</string>
   </property>
   <property name="toolTip">
    <string>CANCEL</string>
   </property>
  </action>
  <action name="action_IDF1">
   <property name="text">
    <string>$IDF1</string>
   </property>
   <property name="toolTip">
    <string>IDENTIFY</string>
   </property>
  </action>
  <action name="action_IDC1">
   <property name="text">
    <string>$IDC1</string>
   </property>
   <property name="toolTip">
    <string>CANCEL</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resource.qrc"/>
  <include location="morpho.qrc"/>
 </resources>
 <connections/>
</ui>
